src/sanitizer.js

```diff
@@ -34,24 +34,27 @@ export const _normalizeConfig = function _normalizeConfig(config) {
     config = {};
   }
 
-  config = Object.keys(config).filter((key) => {
-    return supportedConfigurationLists.has(key);
-  });
-
+  let normalizedConfig = {};
   // TODO https://github.com/mozilla/sanitizer-polyfill/issues/29
   for (let [configurationElementList, elements] of Object.entries(config)) {
-    config[configurationElementList] = elements.map((element) => {
-      return element.toLowerCase();
-    });
+    if (supportedConfigurationLists.has(configurationElementList)) {
+      normalizedConfig[configurationElementList] = elements.map((element) => {
+        return element.toLowerCase();
```

Comment:
Nit: AFAIU you don't need to return inside an arrow-function when you only have one statement.
E.g. `(element) => { element.toLowerCase() }`

Reply:
FWIW, we can/need to do additional filtering.

For allowElements, we can only accept element names that are part of the build-in standard.
A configuration that tries to allow  `script` is invalid and should throw (what _exact_ error we are throwing is currently undefined, so please think of a reasonable Error and file a follow-up :-)

Reply:
> Nit: AFAIU you don't need to return inside an arrow-function when you only have one statement.
> E.g. `(element) => { element.toLowerCase() }`

I gave this a try and it returns a list of `undefined` :(

For example, `config = { "allowElements" : ["TestUpperCase", "alreadylowercase"] }` becomes ` allowElements: [undefined, undefined]` after the map function

Reply:
hm, yeah that's bad. Then, let's not :grin: 

sorry.